packages/integrations/cloudflare/src/utils/image-config.ts

```diff
@@ -35,7 +37,7 @@ export function setImageConfig(
 		default:
 			if (config.service.entrypoint === 'astro/assets/services/sharp') {
 				logger.warn(
-					`The current configuration does not support image optimization. To allow your project to build with the original, unoptimized images, the image service has been automatically switched to the 'noop' option. See https://docs.astro.build/en/reference/configuration-reference/#imageservice`,
+					`The current configuration does not support image optimization. To allow your project to build with the original, unoptimized images, the image service has been automatically switched to the 'passthrough' option. See https://docs.astro.build/en/reference/configuration-reference/#imageservice`,
```

Comment:
A bit of an extra, the option name itself is not `noop` (although yes, it's a non-operational service). Mostly to make things consistent.